common/_util.c

```diff
@@ -0,0 +1,107 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019 Datadog, Inc.
+#include "_util.h"
+#include "datadog_agent.h"
+
+#include <sixstrings.h>
+#include <stdio.h>
+#define MODULE_NAME "_util"
```

Comment:
Could we move this to _util.h and use it in three (like here https://github.com/DataDog/datadog-agent-six/pull/41/commits/6e4bb9c36d39d1c7e1b7eed4c8c65618a664427b, PR abour to get merged).